libraries/AP_Quicktune/AP_Quicktune.h

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#ifndef QUICKTUNE_ENABLED
+ #define QUICKTUNE_ENABLED 1
+#endif
+
+#if QUICKTUNE_ENABLED
+
+#include <AP_Vehicle/AP_Vehicle.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Logger/AP_Logger.h>
+#include <AP_Arming/AP_Arming.h>
+#include <AC_AttitudeControl/AC_AttitudeControl.h>
+#include <GCS_MAVLink/GCS.h>
+#include <RC_Channel/RC_Channel.h>
```

Comment:
Definitel don't need all of these headers in this header.

Move as many as possible to the .cpp file - it reduces compile and re-compile times